library/src/morpheus-master/sass/modules/tool/resources/_resources.scss

```diff
@@ -53,7 +53,7 @@
 			}
 			&.inactive {
 				* {
-					color: #333 !important;
+					color: var(--sakai-text-color-2) !important;
```

Comment:
Do we want to use the disabled colour here?

Reply:
can do